City-of-Helsinki/helsinki-paatokset
# Pp 272 misc frontend fixes

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-272

**Note**
Missing _Muuta etusivun kalenterin elementit kokonaan linkeiksi_ and  _Päättäjät-sivun Luottamushenkilöt-listaa pitää muuttaa siten, että sen nimeksi tulee Valtuuston jäsenet ja siinä näytetään ainoastaan valtuuston jäsenet ja varajäsenet_

**How to test**

Checkout branch and run `drush cim -y; drush cr`

Go to meeting calendar page (/kokouskalenteri) check that the text is now translatable and that there is a spinner when loading the page. The translatable text should appear here https://helsinki-paatokset.docker.so/fi/admin/config/regional/translate. The arrows should now also be visible in mobile view.

Go to some trustee page and check that the translations work in the blue information box.

Go to some policymaker page i.ex. kaupunginvaltuusto and kaupunginhallitus. The texts in the members block should now also be translatable.
The sorting by party should now work so that it's first sorted by party and within the party it is sorted by last name.
Check on some other page than kaupunginvaltuusto that when the deputies are shown, there is visible who's deputy the person is.
